spec/.eslintrc.js

```diff
@@ -2,5 +2,8 @@ module.exports = {
   env: {
     jasmine: true,
     atomtest: true
+  },
+  rules: {
```

Comment:
Just remove this entire section. First of all current ESLint configurations should use the string representation of severity (`2` -> `"error"`) so it's easier to understand when reading it. Secondly the only thing this changes from the [current configuration](https://github.com/airbnb/javascript/blob/fa0a82373eb46ff8e99ccc43ba80c2c18c319e01/packages/eslint-config-airbnb-base/rules/style.js#L344) is the adding of `allowTemplateLiterals`. Currently those are already allowed _when needed_, and disallowed when they are pointless. This change here allows them to be used when they aren't needed which is worse than pointless as template strings require extra processing.